text/086-reduce-barriers-to-contributing-to-stdlib.md

```diff
@@ -0,0 +1,250 @@
+being proven about.
+For example, `Permutation_app : forall A (l m l' m' : list A), Permutation l l' -> Permutation m m' -> Permutation (l ++ m) (l' ++ m')`.
+
+Where applicable the order of parts in a lemma name
+should roughly respect their placement in (the syntax tree of) the
+goal, from outermost to deepest. Part suffixes `_l` and `_r` can be used to indicate
+relative positions of subterms in the AST if relevant.
+For equations where the right-hand side is obvious given the left-hand side, only the left-hand side should be described.
+For example, `Z.add_0_r` describes how to rewrite `z+0`.
+
+Some standard properties are currently identified instead using suffixes.
+Most importantly, `T_rect` is the induction principle for type `T`.
+Other examples include `and_comm: forall A B : Prop, A /\ B <-> B /\ A`.
+Using existing suffixes of kind is not an issue,
+and new ones may be added in cases they are less cumbersome to use than the above style.
+
+Naming, organizational, and aesthetic visions not listed in this section are not a justification to hold back a PR, 
+ut they can be a source for suggestions to both PR content and this section.
+
+## Choosing definitions
+
+If a property or object is straihgtforwardly computable,
+it should be defined through the algorithm for computing it (and as a definition rather than a relation).
+Some users appreciate efficient implementations, but efficiency is not a requirement. 
+easoning-friendly equivalents, constructors, and eliminators should be provided as lemmas right away.
+However, a parametric definitions should not assume that other definitions are formulated in any specific manner,
+so they would have to use `/\` instead of `&&`, for example.
+
+# Compat Strategy
+
+We should make every effort to escape the rigidity trap the stdlib is currently caught in,
+while maintaining the status of stdlib as a reliable piece of infrastructure to depend upon and followed through its development.
+Stability in the sense of avoiding change is not a goal.
+The median change to stdlib will require backwards-compatible changes in multiple CI developments.
+These changes would be routinely merged by maintainers of these developments,
+and co-developed by them if necessary due to complexity of their code, unusual requirements, or otherwise.
+Once-and-over-with strategies for CI-wide changes should be preferred of phased rollouts whenever practical,
+to avoid needless code churn and developer toil in both stdlib and downstream.
+In cases where an atomic replacement is infeasible, but there are strong reasons for making the change,
+a pilot rollout in representative developments should precede an add-deprecate-replace-remove cycle.
+No step in this process should involve waiting for another Coq release, or any specific amount of time --
+if we have ported the CI and are confident that other developments can be ported, the change should proceed without delay.
```

Comment:
With the "No step in this process should involve waiting for another Coq release" part, I'm not sure if what was meant is that we don't have to wait for a release with the deprecation to do the removal. If that's the case, then, it's a change in our compatibility strategy that I don't agree with (or at least, that should be discussed more). However, with the split of the Stdlib, it becomes possible to do more releases than Rocq. So, one could do a Stdlib release quickly just to be able to ship a deprecation and then proceed to the removal immediately in the master branch.